File: Amigo.Tenant.UI.Web/src/app/model/osVersionDTO.ts
import { Component } from '@angular/core';

export class OSVersionDTO {

    platformId: number;
    platformName: string;
    oSVersionId: number;
    versionNumber: string;
    versionName: string;

    constructor(oSVersionId: number, versionName: string) {
        this.oSVersionId = oSVersionId;
        this.versionName = versionName;
    }
}

File: Amigo.Tenant.Application.DTOs/Responses/Security/OSVersionDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Amigo.Tenant.Application.DTOs.Responses.Security
{
    public class OSVersionDTO
    {
        public int PlatformId { get; set; }
        public string PlatformName { get; set; }
        public int OSVersionId { get; set; }
        public string VersionNumber { get; set; }
        public string VersionName { get; set; }

    }
}
